testing/test_terminal.py

```diff
@@ -2312,3 +2343,19 @@ def test_foo():
                 ]
             )
         )
+
+
+def test_raw_skip_reason() -> None:
```

Comment:
Perhaps add a similar test for the `wasxfail` case?

Reply:
Added, thanks.